apply/apply_test.go

```diff
@@ -222,52 +224,80 @@ func TestApplySmokeTest(t *testing.T) {
 	r := require.New(t)
 	fs, _, err := util.TestFs()
 	r.NoError(err)
-	// defer os.RemoveAll(d)
+
+	yml := `
+defaults:
+  owner: foo
+  project: bar
+  terraform_version: 0
+  backend:
+    bucket: baz
+    region: qux
+    profile: quux
+version: 2
+`
+	b, e := yaml.Marshal(yml)
+	r.NoError(e)
+	e = afero.WriteFile(fs, "fogg.yml", b, 0644)
+	r.NoError(e)
+	c, e := v2.ReadConfig([]byte(yml), fs, "fogg.yml")
+	r.NoError(e)
+
+	w, e := c.Validate()
+	r.NoError(e)
+	r.Len(w, 0)
+
+	e = Apply(fs, c, templates.Templates, false)
+	r.NoError(e)
+}
+
+func TestApplySmokeTestJSON(t *testing.T) {
+	r := require.New(t)
+	fs, _, err := util.TestFs()
+	r.NoError(err)
 
 	json := `
-{
-  "defaults": {
-    "aws_region_provider": "reg",
-    "aws_region_backend": "reg",
-    "aws_profile_provider": "prof",
-    "aws_profile_backend": "prof",
-    "aws_provider_version": "0.12.0",
-    "account_id": 789,
-    "infra_s3_bucket": "buck",
-    "project": "proj",
-    "terraform_version": "0.100.0",
-		"owner": "foo@example.com",
-		"tools": {
-			  "travis_ci": {
-						"enabled": true,
-						"aws_iam_role_name": "travis",
-						"id_account_name": "id",
-						"test_buckets": 7
-			  }
+	{
+	  "defaults": {
+		"aws_region_provider": "reg",
+		"aws_region_backend": "reg",
+		"aws_profile_provider": "prof",
+		"aws_profile_backend": "prof",
+		"aws_provider_version": "0.12.0",
+		"account_id": 789,
+		"infra_s3_bucket": "buck",
+		"project": "proj",
+		"terraform_version": "0.100.0",
+		"owner": "foo@example.com"
+	  },
+	  "travis_ci": {
```

Comment:
looks like a bad merge here

Reply:
Not sure what you're referring to

Reply:
defaults.tools.travis_ci and then on the right travis_ci is not nested under tools and not under defaults

Reply:
is that intentional?

Reply:
I assumed it might've been the result of a merge conflict since the format on the right is not a valid v2 format anymore